test/e2e/argo_server_test.go

```diff
@@ -1037,6 +1037,30 @@ spec:
 	})
 }
 
+func (s *ArgoServerSuite) TestArtifactServerArtifactPassing() {
```

Comment:
I feel like we should specify that this checking Archived Workflows specifically, though I do see that some of the tests name it explicitly and others don't 😕

Reply:
Will do.

Reply:
@agilgur5 I changed the name to `TestArtifactServerArchivedWorkflow`. Is this more along the lines of what you were thinking?

Reply:
Works for me. Naming is [hard](https://martinfowler.com/bliki/TwoHardThings.html), especially for this level of specificity 😅